subsys/testsuite/ztest/src/ztest_new.c

```diff
@@ -464,6 +475,26 @@ struct ztest_unit_test *ztest_get_next_test(const char *suite, struct ztest_unit
 	return NULL;
 }
 
+#ifdef CONFIG_ZTEST_DO_THE_SHUFFLE
+void z_ztest_shuffle(void* array,  size_t num_items, size_t elem_size)
```

Comment:
This can be static since it's not externally linked. I think you can remove the `ifdef` above once you set it as static since it'll be removed by the compiler